xml/admin_ceph_upgrade.xml

```diff
@@ -6,1852 +6,633 @@
-     Therefore plan the migration for times when you expect little or no
-     &cephfs; load.
-    </para>
-   </listitem>
-  </itemizedlist>
  </sect1>
- <sect1 xml:id="upgrade-backup">
-  <title>Back Up Cluster Data</title>
 
-  <para>
-   Although creating backups of a cluster's configuration and data is not
-   mandatory, we strongly recommend backing up important configuration files
-   and cluster data. Refer to <xref linkend="cha-deployment-backup"/> for more
-   details.
-  </para>
- </sect1>
- <sect1 xml:id="upgrade-ntp">
-  <title>Migrate from <systemitem class="daemon">ntpd</systemitem> to <systemitem class="daemon">chronyd</systemitem></title>
 
-  <para>
-   &cephos; no longer uses <systemitem class="daemon">ntpd</systemitem> to
-   synchronize the local host time. Instead,
-   <systemitem class="daemon">chronyd</systemitem> is used. You need to migrate
-   the time synchronization daemon on each cluster node. You can migrate to
-   <systemitem>chronyd</systemitem> either
-   <emphasis role="bold">before</emphasis> the cluster, or upgrade the cluster
-   and migrate to <systemitem class="daemon">chronyd</systemitem>
-   <emphasis role="bold">afterward</emphasis>.
-  </para>
+<!-- I think this can go somewhere better in the admin guide -->
 
-  <procedure>
-   <title>Migrate to <systemitem class="daemon">chronyd</systemitem> <emphasis>before</emphasis> the Cluster Upgrade</title>
-   <step>
-    <para>
-     Install the <package>chrony</package> package:
-    </para>
-<screen>&prompt.sminion;zypper install chrony</screen>
-   </step>
-   <step>
-    <para>
-     Edit the <systemitem class="daemon">chronyd</systemitem> configuration
-     file <filename>/etc/chrony.conf</filename> and add NTP sources from the
-     current <systemitem class="daemon">ntpd</systemitem> configuration in
-     <filename>/etc/ntp.conf</filename>.
-    </para>
-    <tip>
-     <title>More Details on <systemitem class="daemon">chronyd</systemitem> Configuration</title>
-     <para>
-      Refer to
-      <link xlink:href="https://documentation.suse.com/sles/15-SP1/html/SLES-all/cha-ntp.html"/>
-      to find more details about how to include time sources in
-      <systemitem class="daemon">chronyd</systemitem> configuration.
-     </para>
-    </tip>
-   </step>
-   <step>
-    <para>
-     Disable and stop the <systemitem class="daemon">ntpd</systemitem> service:
-    </para>
-<screen>&prompt.sminion;systemctl disable ntpd.service &amp;&amp; systemctl stop ntpd.service</screen>
-   </step>
-   <step>
-    <para>
-     Start and enable the <systemitem class="daemon">chronyd</systemitem>
-     service:
-    </para>
-<screen>&prompt.sminion;systemctl start chronyd.service &amp;&amp; systemctl enable chronyd.service</screen>
-   </step>
-   <step>
-    <para>
-     Verify the status of chrony:
-    </para>
-<screen>&prompt.sminion;chronyc tracking</screen>
-   </step>
-  </procedure>
+ <sect1 xml:id="filestore2bluestore">
```

Comment:
SES6 doc have section on that as well, https://documentation.suse.com/ses/6/single-html/ses-deployment/#filestore2bluestore, reference sounds better